tests/integration/api/test_fallback_handler.py

```diff
@@ -0,0 +1,34 @@
+from unittest import mock
+
+import pytest
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from tests.integration.constants import api_url_v1
+
+
+class FallbackException(Exception):
+    pass
+
+
+def raise_exception():
+    raise FallbackException
+
+
+@pytest.mark.django_db
+@mock.patch("aap_eda.api.views.tasks.list_jobs", new=raise_exception)
+def test_debug_unexpected_exception(client: APIClient, settings):
+    settings.DEBUG = True
+    with pytest.raises(FallbackException):
+        client.get(f"{api_url_v1}/tasks/")
+
+
+@pytest.mark.django_db
+@mock.patch("aap_eda.api.views.tasks.list_jobs", new=raise_exception)
+def test_non_debug_unexpected_exception(client: APIClient, settings):
+    settings.DEBUG = False
+    response = client.get(f"{api_url_v1}/tasks/")
```

Comment:
Can you use a different endpoint for the test? This endpoint is not used at all and is going to be removed.

Reply:
Switched to 'projects'.